LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaMessage/LoRaPayloadJoinRequest.cs

```diff
@@ -15,7 +15,7 @@ public class LoRaPayloadJoinRequest : LoRaPayload
         /// <summary>
         /// Gets or sets aka JoinEUI.
         /// </summary>
-        public Memory<byte> AppEUI { get; set; }
+        public JoinEui AppEUI { get; set; }
```

Comment:
Should we use this as an opportunity to rename `AppEUI` to the more conventional `AppEui`?

Reply:
Done in 6339ab6c5009e7292142037060b8e8c60128f308